lib/openapi3.js

```diff
@@ -18,22 +18,35 @@ const dereference = require('reftools/lib/dereference.js').dereference;
 const clone = require('reftools/lib/clone.js').circularClone;
 const swagger2openapi = require('swagger2openapi');
 
+const fs = require('fs').promises;
+
 const common = require('./common.js');
 
 let templates;
 
-function convertToToc(source, data) {
+async function loadMD(sourceSection, destSection) {
+    for (let field in sourceSection) {
+        if (field.startsWith("x-md-")) {
+            //Replacing '-' so that field can be used in dot templates
+            let newFieldName = field.replace("x-md-", "md_");
```

Comment:
Should we replace all `-` with `_` not just the first two? Either with a split/join or a `.replace`.